src/plugins/syscalls/syscalls.cpp

```diff
@@ -378,6 +407,45 @@ static void print_footer(output_format_t format, uint32_t nargs)
     }
 }
 
+static event_response_t win_ret_cb(drakvuf_t drakvuf, drakvuf_trap_info_t* info)
+{
+    syscall_wrapper_t* wrapper = (syscall_wrapper_t*)info->trap->data;
+    syscalls* s = wrapper->sc;
+    int nargs = wrapper->wsc->num_args;
+    uint32_t thread_id = 0;
+
+    if ( !s || !wrapper->args )
+        goto exit;
+
+    if ( info->regs->cr3 != wrapper->target_cr3 )
+    {
+        PRINT_DEBUG("Syscall return callback hit but CR3 (0x%lx) doesn't match target process (0x%lx)\n",
+                    info->regs->cr3, wrapper->target_cr3);
+        goto exit;
+    }
+
+    if ( !drakvuf_get_current_thread_id(drakvuf, info->vcpu, &thread_id) || thread_id != wrapper->target_tid )
+        goto exit;
+
+    print_header(s->format, drakvuf, info);
+    if ( nargs )
+    {
+        print_nargs(s->format, nargs);
+        print_args(s, drakvuf, info, wrapper->wsc, wrapper->args);
+    }
+    print_ret_value(s->format, s->reg_size, info->regs->rax);
+    print_footer(s->format, nargs);
+
+exit:
+    //drakvuf_remove_trap(drakvuf, info->trap, NULL);
+    if (wrapper->args)
```

Comment:
Even if you leave" wrapper->args=null", But one system call will trigger many "win_ret_cb" callbacks, which will result in virtual machine downtime for a long time. How to solve this problem?